src/utils.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * A utilities library.
+ *
+ * @class utils
+ * @singleton
+ */
+
+CKEDITOR.define( function() {
+	return {
+		/**
+		 * Extends one JavaScript object with the properties defined in one or more objects. Existing properties are
+		 * overridden.
+		 *
+		 * @param {Object} target The object to be extended.
+		 * @param {Object} source One or more objects which properties will be copied (by reference) to `target`.
+		 * @returns {Object} The `target` object.
+		 */
+		extend: function( target, source ) {
+			if ( !this.isObject( source ) && !this.isFunction( source ) ) {
```

Comment:
As the author of this code of code I'd say it doesn't make any sense. I must have been tired back then ;)